common/views/components/IIIFViewerPrototype/IIIFViewerPrototype.tsx

```diff
@@ -23,13 +23,14 @@ import { FixedSizeList } from 'react-window';
 import useSkipInitialEffect from '@weco/common/hooks/useSkipInitialEffect';
 import Router from 'next/router';
 import GridViewerPrototype from './GridViewerPrototype';
-import Space from '../styled/Space';
-import Control from '../Buttons/Control/Control';
 import { iiifImageTemplate } from '@weco/common/utils/convert-image-uri';
 import dynamic from 'next/dynamic';
 import LL from '@weco/common/views/components/styled/LL';
 import { PropsWithoutRenderFunction as PaginatorPropsWithoutRenderFunction } from '@weco/common/views/components/RenderlessPaginator/RenderlessPaginator';
 import ImageViewer from '../ImageViewer/ImageViewer';
+import ImageViewerControlsPrototype from './ImageViewerControlsPrototype';
```

Comment:
Can we give these nicer names now?

Reply:
Yeah, good point. At the point of moving anything to be a sibling of the `IIIFViewerPrototype` parent, I guess it would be sensible to remove the `Prototype` suffix.